build_config/webpack.dev.conf.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
         test: /\.(ttf|eot|svg|gif)(\?v=[0-9].[0-9].[0-9])?$/,
         loader: "file-loader?name=[name].[ext]"
       }
+    ],
+    // http://andrewhfarmer.com/aws-sdk-with-webpack/
+    noParse: [
```

Comment:
@adamnfish I asked @NataliaLKB about if client side classes will cover this which is planned for later this year. Think Tools team needs something a bit earlier for our use-case. Welcome to join if you like.